Assets/WizardsCode/Character/_Third Party Demos/Scenes/Ink Demo.unity

```diff
@@ -2370,6 +2370,12 @@ Transform:
   m_Father: {fileID: 1991637198}
   m_RootOrder: 19
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
+--- !u!224 &331439563 stripped
```

Comment:
I updated this scene to work with the new setup. I did not update your other scenes because I was worried about a merge conflict. You basically just put a dialoguebubble in the scene and pass a reference to the ink manager. I moved the choice panel adjacent to the dialogue bubble too